public/js/app/shared/templates/ysearch.html

```diff
@@ -9,16 +9,20 @@
         <div class="y-search-container" ng-show="search.showSearchResults && search.results.length > 0">
             <a class="form-control y-search-results" ng-click="hideSearchResults()"
                ui-sref="base.product.detail( {productId: result.objectID} )" ng-repeat="result in search.results">
-                <div class="attribute">
-                <span class="y-search-result-name text-left"
-                      ng-bind-html="result._highlightResult.name[0].value"></span>
+                <div class="y-search-result-thumbnail">
+                    <img class="y-search-result-image" ng-src="{{result.image}}" />
+                </div>
+                <div class="y-search-result-attributes">
+                    <div class="y-search-result-name text-left"
+                          ng-bind-html="result._highlightResult.name[0].value"></div>
+                    <div class="y-search-result-category text-left">{{'IN' | translate}} {{result.categories.join(', ')}}</div>
+                    <div class="y-search-result-price text-left">{{result.prices[currency] | currency: currencySymbol }}</div>
```

Comment:
When there are for example two sites, US and Germany with $ and € if user changes site still the search results will be showing in previous site currency.